package.json

```diff
@@ -19,21 +19,12 @@
     "app": "gulp app",
     "build": "gulp",
     "postinstall": "gulp postinstall",
-    "preapp": "rm -rf dist",
-    "prebuild": "rm -rf dist",
-    "pres3": "rm -rf dist",
-    "presmoketest": "rm -rf dist",
-    "prestart": "rm -rf dist",
-    "pretest": "rm -rf dist",
-    "pretravis-after-success": "rm -rf dist",
```

Comment:
Some of the tasks here shouldn't be removed just yet since `npm start` is still serving files under `/dist`. i.e., `.html` files are still being generated: https://github.com/mozilla/teach.mozilla.org/blob/develop/lib/build/gulp-index-file-stream.js